### PR TITLE
Issue 296: 2nd editing pass on thr POM file breakdown section 

### LINKED ISSUE #296: Second pass at POM breakdown sections
I took the first pass at the POM breakdown sections in the runtime guides:

http://appdev-stage.openshift.io/docs/wf-swarm-runtime.html#breakdown_pom
http://appdev-stage.openshift.io/docs/vertx-runtime.html#breakdown_pom
http://appdev-stage.openshift.io/docs/spring-boot-runtime.html#breakdown_pom

@inoxx03 would you mind taking a second pass at it?

### LINKED ISSUE #307: [Spring Boot Tomcat Runtime Guide] Appendix B
Looking at number 8 in [Appendix B](http://appdev-stage.openshift.io/docs/spring-boot-runtime.html#breakdown_pom):
`This Maven goal is part of the Spring Boot Tomcat Maven plugin and will repackage the existing jar file that is created during the package goal.`

Actually, the goal is part of `spring-boot-maven-plugin` not Spring Boot Tomcat Maven plugin. In addition, we can explain why is the `repackage` goal there, as stated in the [plugin docs](https://docs.spring.io/spring-boot/docs/current/maven-plugin/repackage-mojo.html):
`Repackages existing JAR and WAR archives so that they can be executed from the command line using java -jar.`

### PR DESCRIPTION
**Pull request by inoxx03**
resolves #296 
also resolves #307

### CODE REVIEW

**Comment by Ladicek** [comment]
CC @RadekKoubsky

**Comment by RadekKoubsky** [comment]
Looks good to me (LGTM).

**Comment by inoxx03** [comment]
@rhoads-zach or @tradej Mind taking a quick look?

File: docs/topics/http-api-wf-swarm-pom.adoc [hunk 1]
```diff
@@ -166,7 +166,8 @@
...
+<7> link:http://arquillian.org/guides/getting_started/#add_the_arquillian_apis[Arquillian APIs] for testing with link:http://junit.org[JUnit].
 <8> link:https://wildfly-swarm.gitbooks.io/wildfly-swarm-users-guide/content/v/2017.6.0/getting-started/tooling/maven-plugin.html[{WildFlySwarm} Maven Plugin] used to create the application link:https://wildfly-swarm.gitbooks.io/wildfly-swarm-users-guide/content/v/2017.6.0/getting-started/concepts.html[uberjar].
-<9> This is the profile for building and deploying the booster to OpenShift. It uses the link:http://fabric8.io/gitbook/mavenPlugin.html[Fabric8 Maven Plugin (FMP)] to build and deploy the application with the S2I Build Process.
-<10> This profile is for running integration tests when the application is running on OpenShift. For example, a test may require a database pod or a ConfigMap value.
+<9> The link:https://maven.apache.org/plugins/maven-war-plugin/[Maven WAR Plugin] collects all artifact dependencies, calsses and resources of your web application and packages them into a WAR file.
```
Comment (Ladicek) [inline docs/topics/http-api-wf-swarm-pom.adoc]: calsses -> classes

File: docs/topics/http-api-spring-boot-pom.adoc [hunk 2]
```diff
@@ -168,7 +168,8 @@
...
-<7> link:https://github.com/spring-projects/spring-boot/tree/v1.4.1.RELEASE/spring-boot-starters/spring-boot-starter-test/pom.xml[Starter POM] for testing Spring Boot applications with libraries including JUnit, Hamcrest and Mockito. 
-<8> This Maven goal is part of the {SpringBoot} Maven plugin and will repackage the existing jar file that is created during the `package` goal.
+<7> link:https://github.com/spring-projects/spring-boot/tree/v1.4.1.RELEASE/spring-boot-starters/spring-boot-starter-test/pom.xml[Starter POM] for testing Spring Boot applications with libraries including JUnit, Hamcrest and Mockito.
+<8> This Maven goal is part of the link:http://docs.spring.io/spring-boot/docs/1.5.4.RELEASE/maven-plugin/index.html[Spring Boot Maven Plugin].  It repackages existing JAR and WAR archives (which are created during the `package` goal) so that they can be executed from the command line using the `java -jar` command.
+You can also specify the `layout=NONE` attribute to package a non-executable JAR file with nested dependencies and no main class.
```
Comment (tradej) [inline docs/topics/http-api-spring-boot-pom.adoc]: Drop "You can also" [minimalism].

File: docs/topics/http-api-vertx-pom.adoc [hunk 3]
```diff
@@ -203,6 +203,6 @@
...
 <7> link:https://vmp.fabric8.io[{VertX} Fabric8 Maven Plugin] used for packaging and deploying {VertX} applications
-<8> link:https://wiki.eclipse.org/M2E_compatible_maven_plugins[Lifecycle metadata] that works with the {VertX} Fabric8 Maven Plugin for building and deploy the application.
+<8> link:https://wiki.eclipse.org/M2E_compatible_maven_plugins[Lifecycle metadata] that works with the {VertX} Fabric8 Maven Plugin for building and deploying the application.
 <9> This is the profile for building and deploying the booster to OpenShift. It uses the link:http://fabric8.io/gitbook/mavenPlugin.html[Fabric8 Maven Plugin (FMP)] to build and deploy the application with the S2I Build Process.
```
Comment (tradej) [inline docs/topics/http-api-vertx-pom.adoc]: Drop "This is".

File: docs/topics/http-api-vertx-pom.adoc [hunk 4]
```diff
@@ -203,6 +203,6 @@
...
-<10> This profile is for running integration tests when the application is running on OpenShift. For example, a test may require a database pod or a ConfigMap value.
```
Comment (tradej) [inline docs/topics/http-api-vertx-pom.adoc]: "The profile for running..." Also, there are two instances of the word "running" in one sentence.

Replace "may" with "can" [IBM Style Guide].

File: docs/topics/http-api-vertx-pom.adoc [hunk 5]
```diff
@@ -203,6 +203,6 @@
...
\ No newline at end of file
+<10> This profile is for running integration tests when the application is running on OpenShift. For example, a test may require a database pod or a ConfigMap value.
```
Comment (tradej) [inline docs/topics/http-api-vertx-pom.adoc]: "The profile for running..." Also, there are two instances of the word "running" in one sentence.

Replace "may" with "can" [IBM Style Guide].

File: docs/topics/http-api-wf-swarm-pom.adoc (hunk 1 above)
Comment (tradej) [inline docs/topics/http-api-wf-swarm-pom.adoc]: "dependencies, classes, and resources of your web application, and packages them..." [Oxford comma; sentence separation]

File: docs/topics/http-api-wf-swarm-pom.adoc [hunk 6]
```diff
@@ -166,7 +166,8 @@
...
+<10> This is the profile for building and deploying the booster to OpenShift. It uses the link:http://fabric8.io/gitbook/mavenPlugin.html[Fabric8 Maven Plugin (FMP)] to build and deploy the application with the S2I Build Process.
```
Comment (tradej) [inline docs/topics/http-api-wf-swarm-pom.adoc]: Drop "This is". [minimalism]

File: docs/topics/http-api-wf-swarm-pom.adoc [hunk 7]
```diff
@@ -166,7 +166,8 @@
...
+<11> This profile is for running integration tests when the application is running on OpenShift. For example, a test may require a database pod or a ConfigMap value.
```
Comment (tradej) [inline docs/topics/http-api-wf-swarm-pom.adoc]: "The profile for running..." Also, there are two instances of the word "running" in one sentence.

Replace "may" with "can" [IBM Style Guide].

**Comment by tradej** [comment]
Comments inline, otherwise LGTM.

**Comment by inoxx03** [comment]
Made another cleanup pass on the POM file breakdown. @tradej & @Ladicek Thanks for the review:)

**Comment by rhoads-zach** [comment]
LGTM, merging.

**Comment by rhoads-zach** [comment]
merged: https://github.com/openshiftio/appdev-documentation/commit/ad373fd9faea0112e580976c01975e30afef408b

**Pull request closed without merging**